File: 01_Basico/02 - Controle_Selecao/Respsotas_em_Python/questao06.py
salarioAtual = float(input("Digite o seu salário atual: "))
aumento = 0
if   salarioAtual <= 280: 
        aumento = (salarioAtual * 20) / 100
elif salarioAtual > 280 and salarioAtual < 700:
        aumento = (salarioAtual * 15) / 100
elif salarioAtual >= 700 and salarioAtual < 1500:
         aumento = (salarioAtual * 10) / 100
elif salarioAtual >= 1500:
        aumento = (salarioAtual * 5) / 100
else:
    print("Opção invalida")

print(salarioAtual + aumento)

File: 01_Basico/02 - Controle_Selecao/Resposta_em_JavaScript/questao06.js
const salarioAtual = 100
let aumento = 0

switch (true){
    case salarioAtual <= 280: 
        aumento = (salarioAtual * 20) / 100
        break
    case salarioAtual > 280 && salarioAtual < 700:
        aumento = (salarioAtual * 15) / 100
        break
    case salarioAtual >= 700 && salarioAtual < 1500:
         aumento = (salarioAtual * 10) / 100
        break
    case salarioAtual >= 1500:
        aumento = (salarioAtual * 5) / 100
    default:
        console.log("Opção invalida")
}
console.log(salarioAtual + aumento)